gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -252,8 +253,8 @@ get_accl_hw_str (const accl_hw key);
  * @brief parse user given string to extract accelerator based on given regex
  */
 extern accl_hw
-parse_accl_hw (const char * accelerators, const char * regex_accl,
-    const char * regex_accl_elem);
+parse_accl_hw (const char * accelerators,
```

Comment:
recommended to define prototype in one line.

Reply:
Updated.